keps/sig-apps/4017-pod-index-label/README.md

```diff
@@ -0,0 +1,879 @@
+This section will be updated accordingly once we align on this.
+
+##### e2e tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+
+We expect no non-infra related flakes in the last month as a GA graduation criteria.
+-->
+
+E2E tests will not provide any additional coverage that isn't already covered by unit + integration tests, 
+since we are simply adding a label, so no e2e tests will be necessary for this change.
+
+### Graduation Criteria
+
+We will release the feature directly in Beta state since there is no benefit in having an alpha release.
+
+#### Beta
+Feature implemented behind the `PodIndexLabel` feature gate.
+Unit and integration tests passing.
+
+#### GA
+Fix any potentially reported bugs.
+
+<!--
+**Note:** Generally we also wait at least two releases between beta and
+GA/stable, because there's no opportunity for user feedback, or even bug reports,
+in back-to-back releases.
+
+**For non-optional features moving to GA, the graduation criteria must include
+[conformance tests].**
+
+[conformance tests]: https://git.k8s.io/community/contributors/devel/sig-architecture/conformance-tests.md
+
+#### Deprecation
+
+- Announce deprecation and support policy of the existing flag
+- Two versions passed since introducing the functionality that deprecates the flag (to address version skew)
+- Address feedback on usage/changed behavior, provided on GitHub issues
+- Deprecate the flag
+-->
+
+### Upgrade / Downgrade Strategy
+
+<!--
+If applicable, how will the component be upgraded and downgraded? Make sure
+this is in the test plan.
+
+Consider the following in developing an upgrade/downgrade strategy for this
+enhancement:
+- What changes (in invocations, configurations, API use, etc.) is an existing
+  cluster required to make on upgrade, in order to maintain previous behavior?
+- What changes (in invocations, configurations, API use, etc.) is an existing
+  cluster required to make on upgrade, in order to make use of the enhancement?
+-->
+
+No changes required to existing cluster to use this feature.
+
+### Version Skew Strategy
+
+<!--
+If applicable, how will the component handle version skew with other
+components? What are the guarantees? Make sure this is in the test plan.
+
+Consider the following in developing a version skew strategy for this
+enhancement:
+- Does this enhancement involve coordinating behavior in the control plane and
+  in the kubelet? How does an n-2 kubelet without this feature available behave
+  when this feature is used?
+- Will any other components on the node change? For example, changes to CSI,
+  CRI or CNI may require updating that component before the kubelet.
+-->
+
+N/A (this feature doesn't impact nodes).
```

Comment:
I'd suggest adding a bit more information, why N/A. Something like:
this feature adds new labels to pods, which doesn't require coordination between various components of the control-plane. In the worst case where a n-1 kube-controller-manager will be creating pods it will create pods w/o the label, and new k-c-m will create with. 
Something like that.

Reply:
done